package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ml-pls",
-  "version": "0.0.4",
+  "version": "0.0.6",
```

Comment:
you shouldn't increase the version number here. It is a separate process to release the package